### ISSUE TITLE
after running yarn upgrade, there are issues with thunk

### ISSUE THREAD

**Issue by portersherman** [portersherman/buddies_fe#8]
Logging in and fetching user data didn't work :/

**Comment by BKDaugherty** [comment]
Did you already fix this? Or add some sort of workaround? I tried to reproduce against a local instance but couldn't.

**Comment by portersherman** [comment]
i'll take another crack at repro'ing it

**Comment by BKDaugherty** [comment]
Don't get me wrong, there are certainly problems with that code lol, was just wondering if there was a specific thing that was broke with the upgrade.

**Comment by portersherman** [comment]
yeah so i just upgraded and restarted the server, and the login flow stopped working? this happened in Chrome (started normally and using `open -na Google\ Chrome --args --user-data-dir=/tmp/temporary-chrome-profile-dir --disable-web-security`)... no helpful error logs though. seems like the backend server is receiving the request though!

**Comment by BKDaugherty** [comment]
Ah okay I was on Firefox. (I'm a real Linux boi now)

I'll take a look and see what's up
